benchmark/assert/deepequal-object.js

```diff
@@ -26,9 +26,9 @@ function createObj(source, add = '') {
 }
 
 function main(conf) {
-  const size = +conf.size;
+  const size = conf.size;
```

Comment:
@BridgeAR Can I remove this line and replace size with conf.size as it is only used in two places(31,34)?

Reply:
If you want to, yes. But I think it is not necessary.